### DISCUSSION TITLE
$5 app or $5 wireless BT keyboard

### DISCUSSION THREAD

**Discussion by arihan-1560795** [AppGround-io/bluetooth-keyboard-and-mouse-support#335] [General]
Aliexpress sells physical portable bluetooth keyboards for >$4; it feels weird spending $5 on an app to do the same. Just dropping my 2c. I acknowledge you've had some success with the app as its sold >10k copies; I just wanted to drop my 2c.

**Comment by Tom-Loader** [comment]
The point is having the ability to do it from your phone, which most people will always have access to. Can pair to multiple devices, switch between them whenever you choose and setup custom layouts. The former some Bluetooth keyboards do the latter they do not. A $5 Bluetooth keyboard from Ali express does not offer the same versatility, you cannot compare the two.
